allennlp/predictors/biaffine_dependency_parser.py

```diff
@@ -105,7 +105,10 @@ def _json_to_instance(self, json_dict: JsonDict) -> Instance:
         """
         spacy_tokens = self._tokenizer.split_words(json_dict["sentence"])
         sentence_text = [token.text for token in spacy_tokens]
-        pos_tags = [token.tag_ for token in spacy_tokens]
+        if self._dataset_reader.use_language_specific_pos: # type: ignore
+            pos_tags = [token.tag_ for token in spacy_tokens]
```

Comment:
It'd be good to have a comment here saying that `token.tag_` in spacy is "fine-grained part of speech", while `token.pos_` is "coarse part of speech".  Is spacy's coarse part of speech even the same as the universal dependencies' pos tag scheme?